src/constants/password.ts

```diff
@@ -8,4 +8,4 @@
 
 import { Enum } from "../types";
 
-export const strengthLevelOrder = Enum("poor", "weak", "average", "good", "excellent");
+export const PASSWORD_STRENGTH_METER = Enum("very-weak", "weak", "fair", "strong", "very-strong");
```

Comment:
could we use camel case instead of kebab case so that it'll be easier to use it?

Reply:
done